src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md

```diff
@@ -90,6 +78,35 @@ After the Pull Request has finally passed the QA validation, it is merged in the
 ![Pull Request process summary](../../img/pull-request-process.png)
 
 
+## Other useful resources
+
+- [What it means to rebase a Pull Request submitted on GitHub](https://dev.to/mathieuks/what-it-means-to-rebase-a-pull-request-submitted-on-github-5717) on dev.to, by Mathieu Ferment
+- [How to resolve a git merge conflict](https://opensource.com/article/20/4/git-merge-conflict) on opensource.com by Brian Breniser
+
+---
+
+_(This article was originally published on our blog: [What Happens To Pull Requests After They Are Submitted](https://build.prestashop.com/news/the-review-process/))_
+d to the next step.
```

Comment:
What do you mean?

Reply:
The content of this page was initially published on https://build.prestashop.com/news/the-review-process/ then moved into the doc. This message informs the user about this history.

Reply:
Oh! I understand now. My PR is messed up.

Reply:
Fixed https://github.com/PrestaShop/docs/pull/946/commits/f22f75fe608e352a959e06a1e69bac823f603256